components/var_exporter.rst

```diff
@@ -177,8 +177,34 @@ populated by using the special ``"\0"`` property name to define their internal v
         "\0" => [$inputArray],
     ]);
 
-Creating Lazy Objects
----------------------
+Creating Lazy Objects on PHP ≥ 8.4
+----------------------------------
+
+Since version 8.4, PHP provides support for lazy objects via the reflection API.
+This native API works with concrete classes. It doesn't with abstracts nor with
+internal ones.
+
+This components provides helpers to generate lazy objects using the decorator
```

Comment:
Should be `component` (without `s`), shouldn't it?

Reply:
yes, fixed while merging. Thanks!